index.bs

```diff
@@ -1993,6 +1995,116 @@ The [=remote end steps=] with |command parameters| are:
 
 </div>
 
+#### The browsingContext.traverseHistory Command ####  {#command-browsingContext-traverseHistory}
+
+The <dfn export for=commands>browsingContext.traverseHistory</dfn> command
+traverses the history of a given context by a delta.
+
+<dl>
+   <dt>Command Type</dt>
+   <dd>
+      <pre class="cddl remote-cddl">
+      BrowsingContextTraverseHistoryCommand = {
+        method: "browsingContext.traverseHistory",
+        params: BrowsingContextTraverseHistoryParameters
+      }
+
+      BrowsingContextReloadParameters = {
+        context: BrowsingContext,
+        delta: int,
```

Comment:
Using a delta to define the number of history entries to move back or forward means that we also need an API to actually retrieve the maximum amount of entries.

Maybe pages might also be interested in to go back/forward to a specific URL.

Reply:
It doesn't mean we need that if there aren't use cases for it. HTTP-based WebDriver just has `/back` and `/forward`. Puppeteer has [goBack](https://pptr.dev/#?product=Puppeteer&version=v9.1.1&show=api-pagegobackoptions) and `goForward`, which [PlayWright](https://playwright.dev/docs/api/class-page/#pagegobackoptions) seems to have kept unchanged. Cypress has [go](https://playwright.dev/docs/api/class-page/#pagegobackoptions) which takes a delta. I can't see any of them having an API to go back/forward to a specific URL or to get a the size of the session history.

I think if those use cases come up later we'd be better off adding an API that returns a list of session history entries, like CDP. Such an PI seems useful for testing the platform itself, but it's hard to argue it's part of the MVP for replacing existing automation tools given none of them provide it.